components/forms/TextArea/TextArea.tsx

```diff
@@ -28,13 +28,7 @@ export const TextArea = (
 
   const [field, meta, helpers] = useField(props);
 
-  const [remainingCharacters, setRemainingCharacters] = useState(0);
-
-  useEffect(() => {
-    if (maxLength) {
-      setRemainingCharacters(maxLength);
-    }
-  }, []);
+  const [remainingCharacters, setRemainingCharacters] = useState(maxLength ? maxLength : 0);
```

Comment:
Just a note that the nullish coalescing operator (??) can also be used to shorten this logic statement.
`useState(maxLength ?? 0)`

Reply:
Love it. Didn't know about the `nullish coalescing operator`.